### PR TITLE
Fix: Unban vendor 

### PR DESCRIPTION
**Pull request by kevinkoech357**
​
## Description
Refactored unban vendor endpoint to follow best API naming conventions and best practices. I also updated the swagger docs and added a short description to the route.

## Related Issue (Link to linear ticket)
None.

## Motivation and Context
Follow best naming conventions and API best practices.
​
## How Has This Been Tested?

​
## Screenshots (if appropriate - Postman, etc):
​
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: super_admin_1/shop/routes.py [hunk 1]
```diff
@@ -503,7 +503,7 @@ def get_banned_vendors(user_id):
 
 # Define a route to unban a vendor
 # WORKS - Documented
-@shop.route("/unban_vendor/<vendor_id>", methods=["PUT"])
+@shop.route("/unban/<vendor_id>", methods=["PUT"])
```
Comment (zxenonx) [inline super_admin_1/shop/routes.py:506]:
```suggestion
@shop.route("/vendors/<vendor_id>/unban", methods=["PUT"])
```

File: super_admin_1/shop/routes.py [hunk 2]
```diff
@@ -503,7 +503,7 @@ def get_banned_vendors(user_id):
...
 @admin_required(request=request)
 def unban_vendor(user_id, vendor_id):
     """
```
Comment (zxenonx) [inline super_admin_1/shop/routes.py:509]: type hints and docstring missing

File: swagger_config.yaml [hunk 3]
```diff
@@ -463,11 +463,12 @@ paths:
                 items:
                   $ref: "#/definitions/Vendor"
 
-  /api/shop/unban_vendor/{vendor_id}:
+  /api/shop/unban/{vendor_id}:
```
Comment (zxenonx) [inline swagger_config.yaml:466]: Check above for the right url

**Review by zxenonx — DISMISSED** [review]
Thanks @kevinkoech357 
I left some comments

File: swagger_config.yaml (hunk 3 above)
Comment (kevinkoech357) [reply swagger_config.yaml]: @zxenonx Made required changes.

**Comment by Freeman-kuch** [comment]
all these except those in swagger docs, have been implementes

**Comment by kevinkoech357** [comment]
> all these except those in swagger docs, have been implementes

@Freeman-kuch type hinting has only been done for params, additionally the change in variable names is confusing. You pass shop_id as a param however when validating and querying db, you still use vendor_id.

**Review by Freeman-kuch — APPROVED** [review]

**Pull request merged**